src/main.js

```diff
@@ -1,16 +1,21 @@
 const template = require('../templates/main.html');
 
 function init () {
+
 	const initialData = window && window.FT && window.FT.storylineData;
 	if (!initialData) return;
 
 	const heatmapSegments = document.getElementsByClassName('n-storylines__heatmap-segment');
 	const backBtns = document.getElementsByClassName('n-storylines__back-btn');
 
 	setupInteraction(initialData);
+	initialData.parent && setupBackBtn(initialData.parent);
```

Comment:
Why is this call to setupBackBtn outside of setupInteraction? Feels a bit odd doing some back button things outside the fn and some inside...